docs/reference/extensions/django-framework.rst

```diff
@@ -43,21 +43,26 @@ application. In the following example we use it to specify ``libpq-dev``:
         # list required packages or slices for your Django application below.
         - libpq-dev
 
+.. warning::
+  You can only use 1 of the dependencies parts at a time.
```

Comment:
Perhaps I missed it, shouldn't we specific explicitly which one we are referring to here?